desktop/src/main/java/bisq/desktop/main/market/offerbook/OfferBookChartViewModel.java

```diff
@@ -276,22 +276,24 @@ private boolean isAnyPricePresent() {
     }
 
     private void updateChartData() {
+
+        // Offer price can be null (if price feed unavailable), thus a null-tolerant comparator is used.
+        Comparator<Offer> offerPriceComparator = Comparator.comparing(Offer::getPrice, Comparator.nullsLast(Comparator.naturalOrder()));
+        // Offer amounts are used for the secondary sort. They are sorted from high to low.
+        Comparator<Offer> offerAmountComparator = Comparator.comparing(Offer::getAmount).reversed();
+
+        var buyOfferSortComparator =
+            offerPriceComparator.reversed() // Buy offers, as opposed to sell offers, are primarily sorted from high price to low.
```

Comment:
(isCrypto ? offerPriceComparator : offerPriceComparator.reversed()) // Buy offers, as opposed to sell offers, are primarily sorted from high price to low.